youtube_dl/extractor/franceculture.py

```diff
@@ -66,4 +71,43 @@ def _real_extract(self, url):
             'alt_title': alt_title,
             'thumbnail': thumbnail,
             'description': description,
+            'display_id': display_id
         }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        return self._extract_infos_from_player(url, video_id)
+
+
+class FranceCultureUrlIE(FranceCultureIE):
+    _VALID_URL = r'https?://(?:www\.)?franceculture\.fr/emission-(?P<id>[^?#]+)'
+    _TEST = {
+        'url': 'http://www.franceculture.fr/emission-les-carnets-de-la-creation-jean-gabriel-periot-cineaste-2015-10-13',
+        'info_dict': {
+            'title': 'Jean-Gabriel Périot, cinéaste',
+            'alt_title': 'Les Carnets de la création',
+            'id': '5093239',
+            'display_id': 'les-carnets-de-la-creation-jean-gabriel-periot-cineaste-2015-10-13',
+            'ext': 'mp3',
+            'timestamp': 1444762500,
+            'upload_date': '20151013',
+            'description': 'startswith:Aujourd\'hui dans "Les carnets de la création", le cinéaste',
+        }
+    }
+
+    # dl url
```

Comment:
This comments are not needed, it's already clear from the code (you can add comment if there's something not obvious or if you have make some relevant choice)